### PR TITLE
documentError

### PR DESCRIPTION
**Pull request by 928PJY**
correct DocFx document errors

### CODE REVIEW

File: Documentation/tutorial/intro_template.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ There are two Markdown files `A.md` and `B.md`, the content for `A.md` is:
 *Renderer*s in [Mustache](http://mustache.github.io) syntax **MUST** end with `.tmpl` extension.
 
 #### Mustache Partials
-[Mustache Partials](http://mustache.github.io/mustache.5.html#partials) is also supported in *DocFX Template System*. **Partials** are common sections of *Renderer* that can be shared by multiple *Renderer* files. **Partials** **MUST** end with `.tmpl.partial`.
+[Mustache Partials](http://mustache.github.io/mustache.5.html#Partials) is also supported in *DocFX Template System*. **Partials** are common sections of *Renderer* that can be shared by multiple *Renderer* files. **Partials** **MUST** end with `.tmpl.partial`.
```
Comment (hellosnow) [inline Documentation/tutorial/intro_template.md]: >Partials [](start = 62, length = 8)

lowercase 'p'artials could also redirect?

Comment (hellosnow) [inline Documentation/tutorial/intro_template.md]: url could be open while bookmark could not redirect

---
In reply to: [95536969](https://github.com/dotnet/docfx/pull/1152#discussion_r95536969) [](ancestors = 95536969)

**Comment by hellosnow** [comment]
:shipit:

**Comment by qinezh** [comment]
better PR name...

**Pull request merged**